### PR TITLE
Infra: Updates ISSUE_TEMPLATE(s)

### PR DESCRIPTION
**Pull request by yardnsm**
### 🔦 Summary

This PR updates our bug report `ISSUE_TEMPLATE`, and also adds a new "Feature Request" template. The structure is very similar to [`yoshi`](https://github.com/wix/yoshi)'s

### CODE REVIEW

File: docs/ISSUE_TEMPLATE/BUT_REPORT.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
```
Comment (NitayRabi) [inline docs/ISSUE_TEMPLATE/BUT_REPORT.md]: Change filename to `BUG_REPORT`

Comment (yardnsm) [reply docs/ISSUE_TEMPLATE/BUT_REPORT.md]: 🙃

File: docs/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+<!---
+  Please add an easy way to reproduce it. Make sure to also provide your WSR (and `yoshi`) version.
+
+  It will be extremely helpful if you could create a simple reproducible example in the following
+  sandbox:
+
+      https://codesandbox.io/s/jj196orvpv?module=%2Fsrc%2FApp%2Findex.js
+  -->
+
+  - **WSR version**: `wix-style-react@6.0.0`
```
Comment (yardnsm) [inline docs/ISSUE_TEMPLATE/BUG_REPORT.md]: leave content empty

File: docs/ISSUE_TEMPLATE/FEATURE_REQUEST.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+---
+
+<!--
+  Thanks for reporting an issue 😄  to `wix-style-react`!
+  Before you submit, please search open / closed issues before submitting, since someone else might
+  have asked the same thing before.
+  -->
+
+# ✨ Feature Request
```
Comment (yardnsm) [inline docs/ISSUE_TEMPLATE/FEATURE_REQUEST.md]: #### Restructure:

- Scope:
	- New component
	- New feature / Behavior Change
- Explanation
- Possible Solution
- Severity

**Pull request merged**